sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/TagsTests.cs

```diff
@@ -326,7 +326,7 @@ private static Activity CreateTestActivity(IEnumerable<KeyValuePair<string, obje
             var eventTimestamp = DateTime.UtcNow;
             var traceId = ActivityTraceId.CreateRandom();
 
-            var parentSpanId = ActivitySpanId.CreateRandom();
+            var parentSpanId = default(ActivitySpanId);
```

Comment:
@mangeg - I am not sure why this change is included.

Reply:
No, that was not supposed to be there. Must have been left in from testing.
Will make a update to revert that change.

Reply:
Thanks!